src/script/standard.cpp

```diff
@@ -438,20 +385,7 @@ CScript GetScriptForMultisig(int nRequired, const std::vector<CPubKey>& keys)
 
 CScript GetScriptForWitness(const CScript& redeemscript)
```

Comment:
Isn't this function itself unnecessary?

Reply:
no, it is scarcely used. I was able to remove it.